db/seeds.rb

```diff
@@ -1,7 +1,317 @@
+  print("\rCreating RMI User #{i}/#{users_per_type}...")
+  RmiUser.create(person(p))
+end
+
+print("\n")
+
+# Asset Manager Users
+asset_managers = people[1 * users_per_type..2 * users_per_type].each_with_index.map do |p, i|
+  print("\rCreating Asset Manager #{i}/#{users_per_type}...")
+  AssetManager.create(person(p))
+end
+
+print("\n")
+
+# Building Operator Users
+building_operators = people[2 * users_per_type..3 * users_per_type].each_with_index.map do |p, i|
+  print("\rCreating Building Operator #{i}/#{users_per_type}...")
+  BuildingOperator.create(person(p))
+end
+
+# Locations
+# -----
+# FORMAT
+# [<name>, <address>, <city>, <state>, <zip>]
+addresses = [
+  ['Unit 1', '2650 Durant Ave', 'Berkeley', :California, 94704],
+  ['Northside Cafe', '1878 Euclid Ave', 'Berkeley', :California, 94709],
+  ['MLK Student Union', '2560 Bancroft Way', 'Berkeley', :California, 94704],
+  ['SMCHS', '22062 Antonio Pkwy', 'Rancho Santa Margarita', :California, 92688],
+  ['Monta Vista HS', '21840 McClellan Rd', 'Cupertino', :California, 95014],
+  ['Archbishop Mitty', '5000 Mitty Way', 'San Jose', :California, 95129],
+  ['The House', '2495 Bancroft Way', 'Berkeley', :California, 94720],
+  ['Googleplex', '1600 Amphitheatre Pkwy', 'Mountain View', :California, 94043],
+  ['Apple Park', '1 Apple Park Way', 'Cupertino', :California, 95014],
+  ['Facebook HQ', '1 Hacker Way', 'Menlo Park', :California, 94025],
+  ['Mozilla HQ', '2 Harrison St', 'San Francisco', :California, 94105],
+  ['Horizons HQ', '450 9th St', 'San Francisco', :California, 94103],
+  ['Little Gem Belgian Waffles', '2468 Telegraph Ave A', 'Berkeley', :California, 94704],
+  ['Gypsy\'s Trattoria Italiana', '2519 Durant Ave', 'Berkeley', :California, 94704],
+  ['People\'s Park', '2556 Haste St', 'Berkeley', :California, 94704],
+]
+
+def generate_location(l)
+  {
+    name: l[0],
+    address: l[1],
+    city: l[2],
+    state: l[3],
+    zip: l[4]
+  }
+end
+
+# Portfolios
+portfolios = asset_managers.map do |a|
+  Portfolio.create!({ name: a.first_name + "'s Portfolio", asset_manager: a })
+end
+
+# Buildings
+print("\n")
+Array(0...portfolios.length).each_with_index do |i, portfolio_index|
+  print("Creating Portfolio #{portfolio_index}/#{portfolios.length - 1}")
+  portfolio = portfolios[i]
+  asset_manager = portfolio.asset_manager
+
+  print("\n")
+  Array(0...addresses.length).each_with_index do |j, building_index|
+    print("\r- Creating Building #{building_index}/#{addresses.length - 1}")
+    location = generate_location(addresses[j])
+    building = Building.new({
+      portfolio: portfolio,
+      name: location[:name],
+      address: location[:address],
+      city: location[:city],
+      state: location[:state],
+      zip: location[:zip]
+    })
+
+    if i.even?
+      building.building_type = large_cube_retail
+    else
+      building.building_type = rockclimbing_eggyolk_inc
+    end
+
+    building.save!
```

Comment:
can you wrap the different types of seeds into functions that are called at the end? this might help readability in the future if seeds should have to change